tools/_venv_common.py

```diff
@@ -91,18 +92,18 @@ def _check_version(version_str, major_version):
     return False
 
 
-def subprocess_with_print(command):
-    print(command)
-    subprocess.check_call(command, shell=True)
+def subprocess_with_print(cmd, env=os.environ, shell=False):
+    print('+ {0}'.format(subprocess.list2cmdline(cmd)) if isinstance(cmd, list) else cmd)
+    subprocess.check_call(cmd, env=env, shell=shell)
 
 
-def get_venv_python(venv_path):
+def get_venv_bin_path(venv_path):
     python_linux = os.path.join(venv_path, 'bin/python')
     if os.path.isfile(python_linux):
-        return python_linux
+        return os.path.join(os.getcwd(), os.path.dirname(python_linux))
```

Comment:
nit: Is the goal here to get an absolute path to the Python binary? If so, maybe we should just use [os.path.abspath](https://docs.python.org/3/library/os.path.html#os.path.abspath).

Reply:
You are right.